documentation/release-notes-2-9.md

```diff
@@ -0,0 +1,2230 @@
+* `RBME` [Rsync Backup Made Easy](https://github.com/schlomo/rbme)
+* `REQUESTRESTORE` Request restore from a human operator
+* `RSYNC` ReaR built-in backup using `rsync` via `rsync` or `ssh` protocol
+* `SESAM` [SEP Sesam](https://wiki.sep.de/wiki/index.php/Bare_Metal_Recovery_Linux)
+* `TSM` IBM Storage Protect / Tivoli Storage Manager / IBM Spectrum Protect
+* `VEEAM` Veeam Backup
+
+ReaR integrates well with Disaster Recovery Linux Manager (DRLM) [drlm.org](https://drlm.org), which can act as a central management tool for ReaR deployments.
+
+## System and Software Requirements
+
+ReaR is written entirely in the native language for system administration: as shell (bash) scripts.
+The intent is that experienced users and system admins can adapt or extend the ReaR scripts
+as needed to make things work for their specific cases.
+
+We need Bash version 4 which is standard on current GNU/Linux systems.
+
+The default backup software used by ReaR is standard GNU Tar.
+
+ReaR is known to work on x86 (32bit and 64bit) and ppc64le architectures.
+ReaR was also ported to ia64 and arm architectures, but these are rarely tested.
+
+## Support
+
+Relax-and-Recover (ReaR) is a Free and Open Source Software project under GPLv3 license.
+
+The creators of ReaR have spend many, many hours in development and support.
+We may give voluntary support, but only as work-life balance allows it.
+Some of the maintainers also provide paid support as a commercial service.
+
+ReaR has a long history (since 2006) and we can therefore only support the latest release version.
+If you have a problem, we urge you to install the latest stable ReaR version
+or the snapshot (development) version available on GitHub before submitting an issue.
+We understand that it is not always possible to install the latest version on hundreds of systems,
+so we are willing to support previous versions of ReaR when you buy a support contract.
+We cannot handle all those various support requests on a voluntary base
+and we give paid projects priority, therefore, we urge our customers
+to buy a support contract for one or more systems.
+Buying a support contract actually buys time for the core developers to work on ReaR.
+
+See the Relax-and-Recover website support [relax-and-recover.org/support](/support)
+
+## Supported and Unsupported Operating Systems
+
+We try to keep our "Test Matrix" wiki pages up-to-date with feedback we receive from the community.
+For example for ReaR 2.9 see <https://github.com/rear/rear/wiki/Test-Matrix-ReaR-2.9>
+
+ReaR 2.9 is supported on the following Linux operating systems:
+
+* Fedora 40, 41 (Silverblue not supported)
+* RHEL 7, 8, and 9
+* RHEL clones (CentOS, Scientific Linux, Alma Linux, Oracle Linux) 7, 8, and 9
+* SLES 15
+* openSUSE Leap 15.x
+* Debian 10 and 11
+* Ubuntu 20.04, 22.04, 24.04
+* Arch Linux / Manjaro
+
+ReaR 2.8 dropped official support for the following Linux operating systems:
```

Comment:
@pcahyna  I think ReaR 2.8 should be ReaR 2.9

Reply:
@pcahyna 
I would like to keep the paragraph as is
because I like explicit information in general and
I think it is valuable information for the user
to see directly in which ReaR version what was dropped
instead of having to reverse-engineer this information
by comparing the lists of supported distribution and
even then there would be no explicit information
that something was dropped (only indirectly by no longer
mentioning that something is supported).